### ISSUE TITLE
Crack doesn't work anymore and xmanager isn't working at all.

### ISSUE THREAD

**Issue by yologamer21022** [Team-xManager/xManager#544]
Cant download any version from the xmanager app and the currently installed isn't working. Not only does the free premium not work but also can't play any music at all.

**Comment by GAMINGCHATS** [comment]
Same issue. Spotify probably fixed the aspect that allowed it to be cracked.

**Comment by i-LJ** [comment]
Hi, this seems to be a problem with certain specific regions. You can change your region by logging in to your browser with a VPN connecting you to another country. Greece and Portugal seem to be the only ones affected by this change as of now, so choose any country but them. Change your account's country via browser, and keep your VPN connection until you log in the app. After you log in you can disconnect and uninstall the VPN and it will work as before. It's also advisable(/required) to reinstall the app from xManager so you have the latest version.

**Comment by Usamabhanbhro** [comment]
Same in Pakistan. Not working

**Comment by Salgueirinho** [comment]
In Portugal isn't working as well

**Comment by Xitee1** [comment]
Still works in Germany currently

**Comment by Undisputed00x** [comment]
Confirming that here in Greece🇬🇷 Xmanager got broken 2 days ago

**Comment by HungNoPro** [comment]
Temporary version available test, at least can listen to music without ads even though the app shows free

**Comment by SyedAounHaiderNaqvi** [comment]
Not working for me in the US

**Comment by GuilhermeTorq** [comment]
I'm located in Portugal. Only with Xmanager it doesn't work (android) but on PC with a different app stills works fine

**Comment by dimostheenis** [comment]
Same for me in Greece, and I tried other sorts of cracked spotify apps and none of them worked either. So it appears to be a change made by spotify

**Comment by Staxville** [comment]
Do we have a list of all the affected countries? I'm going to try the VPN workaround to see whether it solves the issue.

**Comment by Undisputed00x** [comment]
> Do we have a list of all the affected countries? I'm going to try the VPN workaround to see whether it solves the issue.

Changing the profile country using a VPN actually it worked !

**Comment by Staxville** [comment]
> Do we have a list of all the affected countries? I'm going to try the VPN workaround to see whether it solves the issue.

Tried NordVPN to connect to a German server and nothing. Downloaded latest and previous patch and nothing. Tried Turkey, still nothing.

**Comment by andrepetinga** [comment]
> I'm located in Portugal. Only with Xmanager it doesn't work (android) but on PC with a different app stills works fine

Hi. There is an APP to use on PC?? Can you Share it?? Thanks!

**Comment by andrepetinga** [comment]
> Hi, this seems to be a problem with certain specific regions. You can change your region by logging in to your browser with a VPN connecting you to another country. Greece and Portugal seem to be the only ones affected by this change as of now, so choose any country but them. Change your account's country via browser, and keep your VPN connection until you log in the app. After you log in you can disconnect and uninstall the VPN and it will work as before. It's also advisable(/required) to reinstall the app from xManager so you have the latest version.

It worked well!!
Thanks!!!!!

Is the latest version of Spotify, on xManager 8.9.18.512??

It is about 2 months old, but in my phone, this is the latest showing up

**Comment by iCoreAmbition** [comment]
> > Hi, this seems to be a problem with certain specific regions. You can change your region by logging in to your browser with a VPN connecting you to another country. Greece and Portugal seem to be the only ones affected by this change as of now, so choose any country but them. Change your account's country via browser, and keep your VPN connection until you log in the app. After you log in you can disconnect and uninstall the VPN and it will work as before. It's also advisable(/required) to reinstall the app from xManager so you have the latest version.
> 
> It worked well!! Thanks!!!!!
> 
> Is the latest version of Spotify, on xManager 8.9.18.512??
> 
> It is about 2 months old, but in my phone, this is the latest showing up

Yup, It needs an update!

**Comment by GuilhermeTorq** [comment]
> > I'm located in Portugal. Only with Xmanager it doesn't work (android) but on PC with a different app stills works fine
> 
> Hi. There is an APP to use on PC?? Can you Share it?? Thanks!

Yes, don't know if It's allowed to share in anothers project, but it's called SpotX-Official, careful that there was another named SpotX-CLI but that was dropped.

This patch STILL WORKS (in Portugal)

**Comment by Xitee1** [comment]
> > > I'm located in Portugal. Only with Xmanager it doesn't work (android) but on PC with a different app stills works fine
> > 
> > 
> > Hi. There is an APP to use on PC?? Can you Share it?? Thanks!
> 
> Yes, don't know if It's allowed to share in anothers project, but it's called SpotX-Official, careful that there was another named SpotX-CLI but that was dropped.
> 
> This patch STILL WORKS (in Portugal)

What also works is just using the web version with uBlock Origin (which works on any OS, not just windows).

**Comment by Cattn** [comment]
There's also block-the-spot

**Comment by GAMINGCHATS** [comment]
For PC there is spicetify.

**Comment by GAMINGCHATS** [comment]
> > Do we have a list of all the affected countries? I'm going to try the VPN workaround to see whether it solves the issue.
> 
> Tried NordVPN to connect to a German server and nothing. Downloaded latest and previous patch and nothing. Tried Turkey, still nothing.
> 
> EDIT: I just used NordVPN (german server) to change the profile of the account and it worked.
> 1) logout from the app and terminate it.
> 2) set VPN to Germany (or any other country that the app works).
> 3) login from web.spotify.com (not app)
> 4) Edit profile to change it to Germany (or the country the VPN is connected).
> 5) Save changes
> 6) Open app and login again.
> 
> I tried it with 6-7 devices so far and it worked. Didn't need to uninstall the app or update xmanager.

I just tried this with proton vpn and it worked.

**Comment by GAMINGCHATS** [comment]
Also, as far as I can tell, you don't have to have the VPN on always. Once you change the county in your profile you should be able to turn it off. At least on my phone with proton vpn.

**Comment by Staxville** [comment]
The same issue resurfaced today. I did the VPN workaround, set the account to Germany and it worked flawlessly until today.

Any news from other users?

**Comment by GAMINGCHATS** [comment]
Mine works. Account is set to polland and I'm located in Greece.

**Comment by andrepetinga** [comment]
> The same issue resurfaced today. I did the VPN workaround, set the account to Germany and it worked flawlessly until today.
> 
> Any news from other users?

Yes, same issue here... I'm in Portugal.
To make it work again, I just repeated the previous steps:

1. Closed Spotify app (no logout made)
2. Connect to VPN (in my case France).
3. login from web.spotify.com
4. Edit profile and confirmed that France is selected (and yes, it was)
5. Cancel (nothing changed)
6. Opened the spotify app again and voilá, songs are there again

**Comment by enanglous** [comment]
Changed the region to USA about 20 days ago and it worked flawlessly, even had podcasts and audiobooks which are blocked in my home region. The issue came back today and changing the region to Netherlands didn't work. Can someone test out what regions are working currently and provide a list.

Confirmed not working regions:
USA
Netherlands
Pakistan
Greece
Germany
Portugal

**Comment by Xitee1** [comment]
> Changed the region to USA about 20 days ago and it worked flawlessly, even had podcasts and audiobooks which are blocked in my home region. The issue came back today and changing the region to Netherlands didn't work. Can someone test out what regions are working currently and provide a list.
> 
> Confirmed not working regions:
> USA
> Netherlands
> Pakistan
> Greece
> Germany
> Portugal

I'm living in Germany and in a quick test it still worked.

**Comment by Staxville** [comment]
> > Changed the region to USA about 20 days ago and it worked flawlessly, even had podcasts and audiobooks which are blocked in my home region. The issue came back today and changing the region to Netherlands didn't work. Can someone test out what regions are working currently and provide a list.
> > 
> > Confirmed not working regions:
> > USA
> > Netherlands
> > Pakistan
> > Greece
> > Germany
> > Portugal
> 
> I'm living in Germany and in a quick test it still worked.

Can confirm that it works if you change it to Germany. Just did the switch from Poland -> Greece -> Germany.

**Comment by dimostheenis** [comment]
This is just my theory, but I'm afraid they're slowly rolling this out all over the globe. Hope I'm wrong, but the countries are just increasing day by day. This would kill xManager and other alternatives as we know them.

**Comment by Nemillia** [comment]
This still work for France, tried changing my VPN to Germany but somehow ended up in France
